src/Services/Qrda/qrda-export/helper/aggregate_object_helper.rb

```diff
@@ -0,0 +1,150 @@
+module Qrda
+  module Export
+    module Helper
+      module PopulationSelectors
+        def numerator

```

Comment:
@kchapple Can you remove all the ruby files as they are not needed. Also I don't think we need anything in the util directory so may be able to remove entire directory. Unless we need the oid json map files.